api/spec/requests/spree/api/v2/storefront/checkout_spec.rb

```diff
@@ -276,9 +276,9 @@
         let(:params) do
           {
             order: {
-              shipments_attributes: {
```

Comment:
@johannboutet can we please use the previous version - this support multiple shipments scenarios - I think the docs should be updated not the other way around :)